Packager/Models/PodMetadataModels/DigitalAudioFile.cs

```diff
@@ -30,6 +31,10 @@ public override void ImportFromXml(XElement element, IImportableFactory factory)
             Turnover = factory.ToStringValue(element, "turnover");
             Gain = factory.ToStringValue(element, "volume_units");
             Rolloff = factory.ToStringValue(element, "rolloff");
+
+            // additional fields
+            SampleRate = factory.ToStringValue(element, "sample_rate");
+            AnalogTransfer = factory.ToBooleanValue(element, "digital_to_analog");
```

Comment:
I'm a little worried about this conversion here, b/c if the value is not specified, I think it will default to false. Maybe it makes more sense to make this a string or a nullable boolean

Reply:
Talked to Andrew - this is ok given how POD works